.gitignore

```diff
@@ -1,92 +1,2 @@
-build/
-bin/
-lib/
-Debug/
-PreRelease/
-Profile/
-Release/
-ipch/
-__pycache__/
-src/data
-debian/autoreconf*
-debian/files
-debian/*.debhelper.log
-debian/*.substvars
-debian/pioneer*/
-autom4te.cache/
-codedoc/
-nd/Data/
-doxygen/html
-doxygen/latex
-*.vcxproj.user
-*.user
-*.ncb
-*.suo
-*.sdf
-*.opensdf
-*.o
```

Comment:
will this not track all build object files?

Reply:
It should be in `build`.

Reply:
Ah, right. We're on CMake now.